src/test/java/reposense/parser/RepoConfigurationTest.java

```diff
@@ -119,19 +147,58 @@ public void repoConfig_ignoresStandaloneConfig_success()
     @Test
     public void repoConfig_wrongKeywordUseStandaloneConfig_success()
             throws ParseException, GitDownloaderException, IOException {
-        String input = String.format("-config %s", KEYWORD_TEST_CONFIG_FILES);
+        String formats = String.join(" ", CLI_FORMATS);
+        String input = String.format("-config %s -formats %s", IGNORE_STANDALONE_KEYWORD_TEST_CONFIG_FILES, formats);
         CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
 
         List<RepoConfiguration> actualConfigs =
                 new RepoConfigCsvParser(((ConfigCliArguments) cliArguments).getRepoConfigFilePath()).parse();
-        List<RepoConfiguration> authorConfigs =
-                new AuthorConfigCsvParser(((ConfigCliArguments) cliArguments).getAuthorConfigFilePath()).parse();
-        RepoConfiguration.merge(actualConfigs, authorConfigs);
 
         RepoConfiguration actualConfig = actualConfigs.get(0);
         GitDownloader.downloadRepo(actualConfig);
         ReportGenerator.updateRepoConfig(actualConfig);
 
         TestUtil.compareRepoConfig(REPO_DELTA_STANDALONE_CONFIG, actualConfig);
     }
+
+    @Test
+    public void repoConfig_withFormats_ignoreCliFormats() throws ParseException, IOException {
+        String formats = String.join(" ", CLI_FORMATS);
+        String input = String.format("-config %s -formats %s", FORMATS_TEST_CONFIG_FILES, formats);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        List<RepoConfiguration> actualConfigs =
+                new RepoConfigCsvParser(((ConfigCliArguments) cliArguments).getRepoConfigFilePath()).parse();
+        RepoConfiguration.setFormatsToRepoConfigs(actualConfigs, cliArguments.getFormats());
+
+        Assert.assertEquals(1, actualConfigs.size());
+        Assert.assertEquals(CONFIG_FORMATS, actualConfigs.get(0).getFormats());
+    }
+
+    @Test
+    public void repoConfig_withoutFormats_ignoreCliFormats() throws ParseException, IOException {
+        String formats = String.join(" ", CLI_FORMATS);
+        String input = String.format("-config %s -formats %s", WITHOUT_FORMATS_TEST_CONFIG_FILES, formats);
```

Comment:
Actually, now that we have added the `formats` into config file, should we remove `-formats` from cli arguments?

@damithc your opinion on this?